Mac_BSD_resources.py

```diff
@@ -61,7 +61,7 @@ def measure_resources():
 
   # Next, get physical memory (RAM)
   phys_mem = "unable to find size of physical memory"
```

Comment:
Probably not worth changing it now, but it's odd that some of these resource values can be either a string or an integer.